docs/how-to/cluster/migrate.md

```diff
@@ -1,9 +1,11 @@
-(how-to-cluster-replication-migrate-a-cluster)=
-# Migrate from a non-charmed Apache Kafka
+(how-to-cluster-migration)=
+# Migrate from a non-charmed Kafka clusters
```

Comment:
Note: I do remember you said we don't want Apache here (or some similar situation) to highlight that we can do it for other distros/flavours too. But it doesn't feel right to me. Can we discuss that with Giulia please?